qiskit_ibm_runtime/session.py

```diff
@@ -301,15 +302,36 @@ def from_id(
             session_id: the id of the session to be created. This must be an already
                 existing session id.
             service: instance of the ``QiskitRuntimeService`` class.
-            backend: instance of :class:`qiskit_ibm_runtime.IBMBackend` class or
-                string name of backend.
+            backend (DEPRECATED): instance of :class:`qiskit_ibm_runtime.IBMBackend` class or
+                string name of backend. Either ``service`` or ``backend`` must be given.
+
+         Raises:
+            IBMInputValueError: If given `session_id` does not exist. or the backend passed in does
+                not match the original session backend.
 
         Returns:
             A new Session with the given ``session_id``
 
         """
         if backend:
             deprecate_arguments("backend", "0.15.0", "Sessions do not support multiple backends.")
```

Comment:
Not part of this PR, but I found the `Sessions do not support multiple backends` message very confusing, since I can be passing in the correct backend. The message should be something more like `The backend used to open the session will be used`.

Reply:
I'll update the message